app.js

```diff
@@ -0,0 +1,95 @@
+
+let  headlines = []
+let summary = []
+
+function renderHeadlines() {
+  htmlstr="";
+  headlines.map((element, index) => {
+    htmlstr += `<a class='headline-box' id='headline${index}'><img id='headline-picture' src='${element.thumbnail}'><div id='headline-text'>${element.webTitle}</div>`
+  })
+  document.querySelector('.articles').innerHTML=htmlstr;
+  headlines.map((article, index) => {
+  document.querySelector(`#headline${index}`).addEventListener('click', renderSummary(article))
+  })
+}
+
+function renderSummary(article) {
+  return () => {
+    document.querySelector('.articles').innerHTML = `<button class='headline-box'><img id='headline-picture' src='${article.thumbnail}'><div id='headline-text'>'${summary[0].sentences}'</div></button><button class='summary'></button>`
+  }
+}
+
+
+function mockRetrieveHeadlines() {
+    headlines.push(content)
+    headlines.push(content)
+    console.log(headlines[0].webTitle)
+  };
+
+function mockRetrieveSummary () {
+  summary.push(summaries)
+  console.log(summary[0].sentences)
+
+}
+
+function retrieveHeadlines() {
+  const url = "http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http://content.guardianapis.com/uk?show-fields=thumbnail"
```

Comment:
Good to know the show-fields=thumbnail!! I spent a lot of effort go to the article and looked for an image.......

Also, does this link work? I seems not get the right response.